platform/Cargo.toml

```diff
@@ -29,10 +29,11 @@ tree = { path = "packages/tree", default-features = false }
 versioning = { path = "packages/versioning", default-features = false }
 
 # General
-serde = { version = "1", default-features = false }
-thiserror = { version = "1", default-features = false }
 anyhow = "1"
 base64 = { version = "0.21", default-features = false }
+data-encoding = "2.5.0"
```

Comment:
is not there an overlap in the functionalities of the two?

Reply:
Yes, they do indeed overlap but I haven't tested them against each other to see which one would be more performant and/or efficient. The intent here is to use `data-encoding` only for testing purposes until we are sure that we don't want to replace `base64` altogether.